src/main/g8/project/ProjectPlugin.scala

```diff
@@ -0,0 +1,70 @@
+import sbt._
+import sbt.Keys._
+import org.scalafmt.sbt.ScalafmtPlugin.autoImport._
+import sbt.{AutoPlugin, PluginTrigger}
+
+object ProjectPlugin extends AutoPlugin {
+
+  override def trigger: PluginTrigger = allRequirements
+
+  object autoImport {
+
+    lazy val V = new {
+      val fs2            = "0.10.1"
+      val log4cats       = "0.1.0"
+      val logbackClassic = "1.2.3"
+      val circe          = "0.10.0-M1"
+      val freestyleRPC   = "0.14.0"
+      val http4s         = "0.18.15"
+    }
+  }
+
+  import autoImport._
+
+  private lazy val logSettings: Seq[Def.Setting[_]] = Seq(
+    libraryDependencies ++= Seq(
+      "ch.qos.logback"    % "logback-classic" % V.logbackClassic,
+      "io.chrisdavenport" %% "log4cats-core"  % V.log4cats,
+      "io.chrisdavenport" %% "log4cats-slf4j" % V.log4cats
+    ))
+
+
+  lazy val rpcProtocolSettings: Seq[Def.Setting[_]] = Seq(
+    libraryDependencies ++= Seq("io.frees" %% "frees-rpc-client-core" % V.freestyleRPC)
```

Comment:
you need to add here `idlType := "proto",` if not it will be serialized to `avro` by default

Reply:
I'm defining the service in Scala manually as:

```scala
@service(Protobuf) trait PeopleService[F[_]] {

  def getPerson(request: GetPersonRequest): F[GetPersonResponse]

}
```

I'm not interested in generating the proto fields. Actually I'm removing the `sbt-frees-rpc-idlgen` plugin.